lang/en.json

```diff
@@ -898,7 +898,13 @@
     "carousel": {
         "arrowAriaLabel": "Go to slide [NUMBER] of",
         "dotAriaLabel": "Slide number",
-        "activeDotAriaLabel": "active"
+        "activeDotAriaLabel": "active",
+        "playPauseButton": {
+            "play": "Play",
```

Comment:
Do we really need additional nesting level here?

Reply:
For me it is more comfortable when playPauseButton entity is visually separated ad it has several values. But its not critical and if you insist - lets remove nesting

Reply:
Honestly, i do not like the way of using it and if values such as 'carousel.playPauseButton.ariaPlay' could be shorthanded we should do it

Reply:
no problem - done